enzy_htp/_interface/rosetta_interface.py

```diff
@@ -1498,5 +1500,173 @@ def build_cartesian_ddg_engine(
 
         )
 
+
     # endregion
- 
+
+    def add_missing_residues(self, 
+            stru: Structure,
+            missing_residues:List[SeqRes],
+            work_dir:str=None,
+            **kwargs
+        ) -> None: 
+        """Uses the LoopRemodel mover to add missing loop residues to a Structure(). All work is done 
+        to the Structure() in place.
+
+        Args:
+            stru: The Structure() with missing loop Residue()'s.
+            missing_residues: The List[SeqRes] of missing Residues.
+            work_dir: The directory where work will take place. Optional.
+
+        Returns:
+            Nothing.
+        """
+
+        def fix_remodel(stru, seq_res, work_dir):
```

Comment:
Missing type hinting

Reply:
Added in! This method will eventually be phased out but I include it for now.